### PR TITLE
add image to website [fixes #42]

### LINKED ISSUE #42: Replace image on homepage
Preferably logo of Code for Poznan, when it's delivered.

### PR DESCRIPTION
**Pull request by agacix**
Changes a profile.png to Code for Poznan logo on website

### CODE REVIEW

**Comment by OtisRed** [comment]
@magul could you check why Travis has failed? Apparently it run for 10mins but nothing happened, so he quited with fail.

**Comment by magul** [comment]
I re-run and it works fine that time. Don't know what happened there (and we actually don't test anything here, we use Travis just to deploy changes to our live website).

Feel free to merge @OtisRed

**Comment by OtisRed** [comment]
Let's leave it for till next wednesday as I think @agacix wanted to work on the size of the logo.

**Comment by OtisRed** [comment]
Don't mind it. The code was ok. Travis apparently chocked had some moods. @agacix so we`ll leave it hanging until you decide it's ok. You can edit [WIP] - work in progress - tag in the title :)

**Comment by magul** [comment]
There's no your fault here @agacix! Briefly looking on it it looks that our infrastructure has some hiccup.

File: index.html [hunk 1]
```diff
@@ -73,10 +73,7 @@
         <div class="container" id="maincontent" tabindex="-1">
             <div class="row">
                 <div class="col-lg-12">
-                    <img class="img-responsive" src="img/profile.png" alt="code for poznan">
-                    <div class="intro-text">
-                        <h1 class="name">Code for Poznan</h1>
-                    </div>
+                    <img class="img-responsive cfp-logo" src="img/logo codeforpoznan.svg" alt="code for poznan">
```
Comment (OtisRed) [inline index.html]: @agacix have you commited `img/logo codeforpoznan.svg` file? I don't see it in the pull request

**Pull request merged**